.editorconfig

```diff
@@ -3,7 +3,7 @@ root = true
 [*]
 charset = utf-8
 trim_trailing_whitespace = true
-indent_style = tab
+indent_style = space

```

Comment:
Thanks for catching this - I started with a forked boilerplate repo so this was out of sync with my editor apparently.